event/src/unvalidated/event.rs

```diff
@@ -92,16 +107,17 @@ where
         }
     }
 
+    // TODO : Returns the cid no option, update the comment
     /// Returns the 'id' field of the event, which is the Cid of the stream's init event.
     /// If this event *is* the init event, then it doesn't know its own Cid and returns None.
```

Comment:
Update this comment to indicate we always return the cid because and init event does know its cid